languages/python/flipper/__init__.py

```diff
@@ -1,5 +1,5 @@
-from lf import *
+from .lf import *
```

Comment:
What does this `.` do here?

Reply:
That's the new syntax for explicit relative imports. It means import from the current package.